examples/build-package/main.tf

```diff
@@ -426,6 +426,7 @@ module "lambda_layer_poetry" {
     {
       path           = "${path.module}/../fixtures/python-app-poetry"
       poetry_install = true
+      poetry_tmp_dir = "${path.root}/builds/"
```

Comment:
Could you change this line similar to the bellow existing in examples? https://github.com/terraform-aws-modules/terraform-aws-lambda/blob/abd5a15fd73ddd6ed36569b94f5f7667d73d5f4a/examples/build-package/main.tf#L40
I think it's better to keep the `builds/` folder with just the module generated content and doesn't give such advice in examples to use it for other purposes.

Reply:
updated